### DISCUSSION TITLE
Weekly Meeting (2023-07-23)

### DISCUSSION THREAD

**Discussion by drafish** [PlanckerLabs/remix-cc#38] [Meeting of Remix CC]
### 用 issue 来管理周会
之前一直是在 repo 中提交 md 文件来记录周会。

这样做有一个缺点，就是每次都得走一个代码提交的流程。项目组里多个人都要提交的话，还得先 pull 别人的 commit ，然后再自己 commit 。虽然可以用 PR 来管理，但过程太过繁琐。

现在我们改用 issue 来记录周会，周会议程作为 issue 的主体。每个人做了什么，就在 issue 下面评论。

点击 issue 右侧的 label ，即可查看这次以及以后的所有周会。

https://github.com/PlanckerLabs/remix-cc/issues?q=label%3Aweekly-meeting+

### 声誉激励
![image](https://github.com/PlanckerLabs/remix-cc/assets/18543527/03c37442-252f-4710-bb47-bbdf058fd123)
Remix 官网有一栏专门展示团队信息，顺便还提了下赞助方。我们可以在这一栏的下面新增一栏，用于展示 CC 团队的信息，以及普朗克和 GCC 对我们的支持。以后这个镜像站就是我们对外的一个门面。我们对外宣传项目的同时也在帮项目组成员和赞助方做宣传。

目前有两个小问题要解决
1. 怎么用真人照片生成卡通头像。这事儿没人做的话就组长来研究下
2. 项目组成员最好都写个自我介绍，像 Rory 的 github 主页那样就很好

### 事后回溯
目前项目资金来源于三方
1. Remix（较少）
2. GCC（较多）
3. 普朗克（兜底）

目前资金申请方式都是事前申请。这种方式有几个缺点
1. 要对未来做预判，这不仅会产生很高的掰扯成本，而且预算还估不准，估多估少都不好
2. 先定预期再做事，项目组就会奔着预期去做，虽然产出会有保障，但很可能会被限制住上限
3. 太过依赖组长这个中心化的节点，这会导致组长的管理压力比较大，同时会限制组员的主观能动性

鉴于这几个缺点，我想把资金申请方式从事前申请改成事后回溯。

事后回溯并不是完全没有计划，至少项目组内要有个大概的方向。每个组员要自己决定下个阶段要做哪些事，等到有阶段性产出后要自己写产出报告，要写明产出是什么，花了多少时间，希望获得多少赞助。对有些可能会引起争论的点要做出说明，否则可能会影响赞助方的判断，从而导致得不到赞助。当然，如果有人特别豪横，根本不在意有没有赞助，也可以不写产出报告。组长只需要把组员的产出报告整理一下，然后给赞助方做个展示。赞助方内部会讨论并投票，对他们认可的产出进行捐赠。

这里会存在一个风险，就是有些产出可能得不到赞助方的认可。我的意思是，这个风险由我们项目组自己承担，并且分散到每个组员的头上。我们不能因为赞助方认可一件事我们才去做，他们不认可我们就不去做。我们要有自己的判断，我们自己决定要做的事，不管有没有赞助，我们都要去做。至于能不能拿到赞助，能拿到多少，这个随缘。只有放下对钱的执念，才能更好地专注于事。把事做好了，钱自然会有。

简单来说，就是牺牲稳定性换取更大的自由度和创造性。

不过，这个季度因为已经定了，那我们就按现状先跑着。下个季度，我们可以考虑下转成事后回溯。

**Comment by drafish** [comment]
### DJQ
1. 完成了 Remix 官网的国际化和中文翻译，并且向官方提交了 PR 。国内镜像站也做了更新 https://remix-project.cn/
2. 在 GCC 周会上明确了捐赠执行方案，预计是在一周内拨款。详见 https://metaforo.io/g/gcc/thread/remix-china-45304
3. 和 Remix 那边开会同步了我们近期的一些进展。详见 https://github.com/PlanckerLabs/remix-cc/issues/15

**Comment by gongzishuye** [comment]
### Mocca
1. 修正remix docs翻译中markdown语法错误。完成30%。💬 https://docs.remix-project.cn/index.html
